src/README.md

```diff
@@ -0,0 +1,53 @@
+# WooCommerce `src` files
+
+This directory is home to new WooCommerce class files under the \Automattic\WooCommerce\ namespace using PSR-4 file naming. This is to take full advantage of autoloading.
+
+Currently, these classes have a PHP 5.6 requirement. No required core classes will be added here until this PHP version is enforced. If running an older version of PHP, these class files will not be used.
+
+## Installing Composer
+
+Composer is used to generate autoload class-maps for the files here. The stable release of WooCommerce comes with the autoloader, however, if you're running a development version you'll need to use Composer.
```

Comment:
Guessing we should update the general README and CONTRIBUTING documentation to add in similar instructions

Reply:
Yes, I'll add this to the todo list on main PR